client/lib/faux-plugins.js

```diff
@@ -224,7 +235,7 @@ module.exports = function(roomName) {
         })
         .filter(Boolean)
 
-      var playRe = /!play [^?]*\?v=([-\w]+)/
+      var playRe = /!play [^?]*\?v=([-\w]+)(&t=([0-9hms]+))?( .*)?/
```

Comment:
What purpose does the added `( .*)?` serve? Matching extra stuff at the end of the URL? This regex is complicated enough at this point that a comment explaining each group would be helpful!